### DISCUSSION TITLE
Document that Alpine doesn't support MariaDB

### DISCUSSION THREAD

**Discussion by lukbukkit** [dani-garcia/vaultwarden#1512] [General]
Hello,

while installing bitwarden_rs on my server via Docker I ran into the issue that the [`alpine` image](https://github.com/dani-garcia/bitwarden_rs/blob/7436b454db5734a225afa88db96fcb62b187bb4d/docker/amd64/Dockerfile.alpine) doesn't support MariaDB.

I was a little bit confused because I've followed the guide [Which container image to use](https://github.com/dani-garcia/bitwarden_rs/wiki/Which-container-image-to-use) on the wiki. It states that both versions of images are functionally the same. After I found and read the [blog post](https://bitwardenrs.discourse.group/t/note-update-to-the-latest-version-before-reporting-an-issue/364) on the forum, I did understand that indeed the `alpine` image does not support MariaDB.

Can I add a notice to the following wiki page to clear the confusion for other users?
https://github.com/dani-garcia/bitwarden_rs/wiki/Which-container-image-to-use
https://github.com/dani-garcia/bitwarden_rs/wiki/Using-the-MariaDB-%28MySQL%29-Backend

Thanks for your this wonderful project.

**Comment by jjlin** [comment]
I updated the first page, but feel free to update the other.

**Comment by BlackDex** [comment]
I'm working on creating static binaries which do support all databases, and could in theory even run in from scratch containers.
Laying some final hands on it.

It's either that, or we could try to build without static on alpine, but i had some weird issues regarding that and all databases.

**Reply by jjlin** [reply]
Static binaries for each release would be pretty useful (as commonly done for Go-based programs). Running from `scratch` might be a novelty, but for most people, I think such containers would be annoying to use in practice.
